src/controllers/user/routes.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express';
+import user from './Controller';
+
+const router: Router = Router();
+
+router.get('/', user.get);
+router.post('/', user.post);
+router.put('/:id', user.put);
```

Comment:
pass id in request body instead of params for put route.

Reply:
changes done